docs/fundamentals/code-analysis/quality-rules/ca2109.md

```diff
@@ -24,11 +24,6 @@ ms.author: gewarren
 
 A public or protected event-handling method was detected.
 
-> [!NOTE]
-> This rule has been deprecated. It last shipped with Microsoft.CodeAnalysis.Analyzers v3.3.0.
```

Comment:
This rule doesn't ship in `Microsoft.CodeAnalysis.Analyzers`. It ships in `Microsoft.CodeAnalysis.NetAnalyzers`.